plugins/source/aws/resources/services/inspector2/coverages.go

```diff
@@ -0,0 +1,54 @@
+package inspector2
+
+import (
+	"context"
+	"github.com/apache/arrow/go/v14/arrow"
+	"github.com/aws/aws-sdk-go-v2/service/inspector2"
+	"github.com/aws/aws-sdk-go-v2/service/inspector2/types"
+	"github.com/cloudquery/cloudquery/plugins/source/aws/client"
+	"github.com/cloudquery/plugin-sdk/v4/schema"
+	"github.com/cloudquery/plugin-sdk/v4/transformers"
+)
+
+func Coverages() *schema.Table {
+	tableName := "aws_inspector2_coverages"
+	return &schema.Table{
+		Name: tableName,
+		Description: `https://docs.aws.amazon.com/inspector/v2/APIReference/API_ListCoverage.html
```

Comment:
Link to the structure doc instead
```suggestion
		Description: `https://docs.aws.amazon.com/inspector/v2/APIReference/API_CoveredResource.html
```